bases/computer_lib.py

```diff
@@ -216,7 +217,16 @@ def change_text(self, button_index):
                 ShowProgress.activateProgressScreen('loading',3)
                 import dj_lib
                 dj_lib.enable()
-                VS.loadGame(self.picker_screen.items[self.picker_screen.selection].data)
+                game = self.picker_screen.items[self.picker_screen.selection].data
+                if not is_utf8(game):
+                    alert = f'  {game} is not UTF-8, convert it before loading'
+                    print(alert)
+                    self.alert_text = GUI.GUIStaticText(self.guiroom, 'txt_screen', alert,
+                        GUI.GUIRect(60, 90, 657, 100), color=GUI.GUIColor(255, 255, 255), bgcolor=GUI.GUIColor(0, 0, 0))
```

Comment:
This needs testing at different resolutions/window-sizes

Reply:
Testing should be easy, as the graphic elements (alert included) react dynamically to window resizing, try it yourself. Haven't tested full screen modes, but they should behave the same way.
As for the needed changes, feel free to apply any you see fit, included the updated check call via Director, e.g.
```
import Director
...
if not Director.isUtf8SaveGame(game):
```

Reply:
The updated alert geometry had been lost in revisions, reinstated; also, check replaced with the C++ Director one. Tests welcome!